myapp/app/controllers/tasks_controller.rb

```diff
@@ -30,18 +27,18 @@ def create
 
   # タスク詳細画面
   def show
-    @task = Task.includes(:user).find(params[:id])
+    @task = Task.find_by(id: params[:id], user_id: login_user.id)
```

Comment:
ここも関連付けを利用すると良いと思います。
```suggestion
    @task = login_user.tasks.find(params[:id])
```

Reply:
ありがとうございます！
それを使う頭がまだ定着していないので、レビューをする中で落とし込んでいきます。